### PR TITLE
Page/reports

### PR DESCRIPTION
**Pull request by SantoshPhaiju**
fixing merge error.

### CODE REVIEW

File: src/components/reports/ReportCard.tsx
Comment (AbhishekSunar01) [inline src/components/reports/ReportCard.tsx:1]: dont use hex color use text-primary instead

Comment (SantoshPhaiju) [reply src/components/reports/ReportCard.tsx:1]: k ma

Comment (SantoshPhaiju) [reply src/components/reports/ReportCard.tsx:1]: text-primary design sanga mildeina

Comment (SantoshPhaiju) [reply src/components/reports/ReportCard.tsx:1]: mailey #14181f colour tapai ko configuration ma vetayena primary colour blue xa muted, destructive, secondary, accent milexaina tapai aafai milaunu tyo

**Review by AbhishekSunar01 — APPROVED** [review]
Great

**Pull request merged**